content/en/images.md

```diff
@@ -31,7 +31,7 @@ Figures MUST be referenced using HTML
 * In a GitHub repository, each technical document should have its own "images" folder.
 * The location of the figure is controlled by the src parameter
 * See Creation of `.svg` Images
-    * Please follow these guidelines when creating a new image from scratch
+    * Please follow these [`guidelines`](https://docs.google.com/document/d/194oEy2GrUdLafICeKUxPVH8zX9okZd_D/edit?usp=sharing&ouid=112978134087833973902&rtpof=true&sd=true) when creating a new image from scratch.
```

Comment:
@gocadimic, the problem with that link is that the document is restricted to only SH staff. We need to download a PDF and upload it to the repository. Also, add a link inside the document to the link in GDrive. People won't be able to download it, but at least we know where the document is, say, six months from today.

Reply:
Resolved in PR #26